commands/system/largest-ram-process.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Largest RAM Process
+# @raycast.mode inline
+# @raycast.refreshTime 1m
+# @raycast.author Caleb Stauffer
+# @raycast.authorURL https://github.com/crstauf
+# @raycast.description Report process with greatest system RAM usage.
+# @raycast.packageName System
+# @raycast.icon 📈
+
+output=$(ps -exco rss,comm | sort -k 1 -n -r | head -n 1)
+bytes=$(echo "$output" | awk -F " " '{ st = index($0," ");print $1}')
+cmd=$(echo "$output" | awk -F " " '{ st = index($0," ");print substr($0,st+1)}')
+
+gigabytes=$(echo "scale=3; $bytes / 1024 / 1024" | bc -l)
```

Comment:
I assume `scale` are the digits after the decimal point? Let's make it 1. This should be enough and aligns with how the Activity Monitor displays it.
```suggestion
gigabytes=$(echo "scale=1; $bytes / 1024 / 1024" | bc -l)
```

Reply:
I would also suggest to show process that consume less than 1 GB of RAM as megabytes, e.g. 834.6 MB. This would align with the Activity Monitor. 

Something along these lines:
```suggestion
if [ $bytes -le 1024 ]; then
  scaled_bytes=...
el
  scaled_bytes=...
fi

formatted_bytes=$(echo "scale=1; $scaled_bytes" | bc -l)
```
  formatted_bytes=$(echo "scale=1; $bytes / 1024 " | bc -l)
  formatted_bytes=$(echo "scale=1; $bytes / 1024 / 1024" | bc -l)

```

Reply:
@thomaspaulmann Updated; set to one decimal place for megabytes and two decimal places for gigabytes (to match Activity Monitor https://cloudup.com/coGkuBW6PUb).